usr/share/rear/conf/default.conf

```diff
@@ -848,6 +848,9 @@ BACKUP_UMOUNTCMD=
 # because when OUTPUT_URL is set OUTPUT_OPTIONS does not inherit the BACKUP_OPTIONS value
 # cf. https://github.com/rear/rear/issues/2753
 OUTPUT_URL=
+# Filter for RESULT_FILES to be uploaded with shell wildcard 
+# e.g. OUTPUT_URL_FILES_FILTER="*.iso" to upload only iso files
+OUTPUT_URL_FILES_FILTER=
```

Comment:
We do not have any `*_FILTER` variable yet, but we have some variables called `*_PATTERNS`. For consistency, would it make sense to change it to match?
The variables are `FILES_TO_PATCH_PATTERNS` and `WRITE_PROTECTED_FS_LABEL_PATTERNS`. (One contains a space separated list of patterns and the other an array of patterns, which is itself inconsistent. I prefer the array myself. )